src/script/calling/mlsConference.ts

```diff
@@ -104,11 +104,24 @@ export const subscribeToEpochUpdates = async (
 
   const forwardNewEpoch = async ({groupId, epoch}: {groupId: string; epoch: number}) => {
     if (groupId !== subconversationGroupId) {
-      return;
+      const parentConversationId = await mlsService.findConversationIdByGroupId?.(groupId);
```

Comment:
I'm not sure this is not too heavy as `forwardNewEpoch` will get called every time any of the group will introduce new epoch... `findConversationIdByGroupId` does a `.find()` on the list of stored conversations. Other approach would be to store `groupId` -> `conversationId` map.